docs/ww-manual/resident-experience.md

```diff
@@ -65,12 +85,58 @@ The resident will then be shown a summary of the subscription details before con
 
 <img alt="Subscription summary" src="/assets/img/ww-user-guide/WasteWorks-subscription-summary.png" class="admin-screenshot" />
 
-The resident will be redirected to your chosen payment provider as part of the WasteWorks workflow.
+The resident will be redirected to your chosen payment provider to complete the subscription as part of the WasteWorks workflow.
+
+### Booking a bulky waste collection
+
+**Please note:** The process for setting up, renewing or cancelling bookings/subscriptions will differ depending on your processes and/or the payment provider you are using.
+
+From the main ‘Your bin days’ page, residents can select a bulky waste collection.
+
+From here, users will be shown some information about the booking process to read before they begin.
+
+Next they will be asked to provide their contact details, before selecting from a list of available collection dates provided by your integrated in-cab system.
+
+<img alt="Available dates for collection" src="/assets/img/ww-user-guide/WasteWorks available dates for collection.png" class="admin-screenshot" />
+
+Users will then be asked for the necessary information about the item(s) for collection. This includes being able to optionally upload photos.
 
-**The process for setting up, renewing or cancelling subscriptions for users will differ depending on the payment provider you are using.**
+<img alt="Bulky waste collection" src="/assets/img/ww-user-guide/WasteWorks bulky waste collection.png" class="admin-screenshot" />
+
+A summary of costs associated with the collection will then be provided.
+
+Following this, usesrs will be asked to provide information in a free text box on where the item(s) will be left to help your waste crew locate it/them on the day of collection. Photos can optionally be provided. 
+
+Finally, the user will be shown a summary of the collection information provided before they are redirected to your payment provider to complete the booking, which is incorporated into the workflow. 
+
+### Booking a small items waste collection
```

Comment:
This is the same as bulky - do we want to have it separately? It's only for brent

Reply:
Yes, just for the sake of spoon feeding clarity

Reply:
So to clarify, yes, we want it to be separate